---
# Documentation: https://sourcethemes.com/academic/docs/managing-content/

title: Stability of DNA methylation and chromatin accessibility in structurally diverse
  maize genomes
subtitle: ''
summary: ''
authors:
- Jaclyn M Noshay
- Zhikai Liang
- Peng Zhou
- Peter A Crisp
- Alexandre P Marand
- Candice N Hirsch
- Robert J Schmitz
- Nathan M Springer
tags: []
categories: []
date: '2021-08-01'
lastmod: 2022-06-01T22:55:52+10:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2022-06-01T12:55:52.145162Z'
publication_types:
- '2'
abstract: Accessible chromatin and unmethylated DNA are associated with many genes
  and cis-regulatory elements. Attempts to understand natural variation for accessible
  chromatin regions (ACRs) and unmethylated regions (UMRs) often rely upon alignments
  to a single reference genome. This limits the ability to assess regions that are
  absent in the reference genome assembly and monitor how nearby structural variants
  influence variation in chromatin state. In this study, de novo genome assemblies
  for four maize inbreds (B73, Mo17, Oh43, and W22) are utilized to assess chromatin
  accessibility and DNA methylation patterns in a pan-genome context. A more complete
  set of UMRs and ACRs can be identified when chromatin data are aligned to the matched
  genome rather than a single reference genome. While there are UMRs and ACRs present
  within genomic regions that are not shared between genotypes, these features are
  6- to 12-fold enriched within regions between genomes. Characterization of UMRs
  present within shared genomic regions reveals that most UMRs maintain the unmethylated
  state in other genotypes with only âˆ¼5% being polymorphic between genotypes. However,
  the majority (71%) of UMRs that are shared between genotypes only exhibit partial
  overlaps suggesting that the boundaries between methylated and unmethylated DNA
  are dynamic. This instability is not solely due to sequence variation as these partially
  overlapping UMRs are frequently found within genomic regions that lack sequence
  variation. The ability to compare chromatin properties among individuals with structural
  variation enables pan-epigenome analyses to study the sources of variation for accessible
  chromatin and unmethylated DNA.
publication: '*G3 GenestextbarGenomestextbarGenetics*'
url_pdf: https://doi.org/10.1093/g3journal/jkab190
doi: 10.1093/g3journal/jkab190
---
