---
# Documentation: https://sourcethemes.com/academic/docs/managing-content/

title: 'Decoding the sorghum methylome: understanding epigenetic contributions to
  agronomic traits'
subtitle: ''
summary: ''
authors:
- 3_Ulduz
- Emma Mace
- David Jordan
- Peter A. Crisp
tags: []
categories: []
date: '2022-02-01'
lastmod: 2022-06-01T22:55:52+10:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2022-06-01T12:55:52.238809Z'
publication_types:
- '2'
abstract: DNA methylation is a chromatin modification that plays an essential role
  in regulating gene expression and genome stability and it is typically associated
  with gene silencing and heterochromatin. Owing to its heritability, alterations
  in the patterns of DNA methylation have the potential to provide for epigenetic
  inheritance of traits. Contemporary epigenomic technologies provide information
  beyond sequence variation and could supply alternative sources of trait variation
  for improvement in crops such as sorghum. Yet, compared with other species such
  as maize and rice, the sorghum DNA methylome is far less well understood. The distribution
  of CG, CHG, and CHH methylation in the genome is different compared with other species.
  CG and CHG methylation levels peak around centromeric segments in the sorghum genome
  and are far more depleted in the gene dense chromosome arms. The genes regulating
  DNA methylation in sorghum are also yet to be functionally characterised; better
  understanding of their identity and functional analysis of DNA methylation machinery
  mutants in diverse genotypes will be important to better characterise the sorghum
  methylome. Here, we catalogue homologous genes encoding methylation regulatory enzymes
  in sorghum based on genes in Arabidopsis, maize, and rice. Discovering variation
  in the methylome may uncover epialleles that provide extra information to explain
  trait variation and has the potential to be applied in epigenome-wide association
  studies or genomic prediction. DNA methylation can also improve genome annotations
  and discover regulatory elements underlying traits. Thus, improving our knowledge
  of the sorghum methylome can enhance our understanding of the molecular basis of
  traits and may be useful to improve sorghum performance.
publication: '*Biochemical Society Transactions*'
url_pdf: https://doi.org/10.1042/BST20210908
doi: 10.1042/BST20210908
---
